gum/backend-libdwarf/gumsymbolutil-libdwarf.c

```diff
@@ -627,6 +714,9 @@ gum_symbol_util_deinitialize (void)
   g_hash_table_unref (gum_function_addresses);
   gum_function_addresses = NULL;
 
+  g_hash_table_unref (gum_address_symbols);
+  gum_address_symbols = NULL;
```

Comment:
Should be moved above `gum_function_addresses`, as these are destroyed in reverse order relative to the order they're declared in.